src/engine/CodeValidator.v3

```diff
@@ -1087,6 +1087,53 @@ class CodeValidator(extensions: Extension.set, limits: Limits, module: Module, e
 				I64_ATOMIC_RMW_CMPXCHG	=> check_MEMARG(opcode, 8);
 				I32_ATOMIC_RMW8_CMPXCHG_U	=> check_MEMARG(opcode, 1);
 				I32_ATOMIC_RMW16_CMPXCHG_U	=> check_MEMARG(opcode, 2);
+				CONT_NEW => {
+					var cont_decl = parser.readCont();
+					popE(ValueTypes.Ref(true, cont_decl.sig));
+					// TODO[ss]: nullable = false as per proposal doc. Confirm/ask this.
+					val_stack.push(ValueTypes.Ref(false, cont_decl));
```

Comment:
I think that's right here; this should produce a non-null reference to a continuation.